learntools/machine_learning/nbs/ch2.ipynb

```diff
@@ -0,0 +1,184 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This exercise will test your ability to read a data file and understand statistics about the data.\n",
+    "\n",
+    "In later exercises, you will techniques to filter the data, build a machine learning model, and iteratively improve your model.\n",
+    "\n",
+    "The course gives examples using data from Melbourne. To ensure you can apply these techniques on your own, you will have to apply them to a new dataset (with house prices from Iowa).\n",
+    "\n",
+    "The exercises use a \"notebook\" coding environment.  In case you are unfamiliar with notebooks, we have a [90-second intro video](https://www.youtube.com/watch?v=4C2qMnaIKL4).\n",
+    "\n",
+    "# Exercises\n",
+    "\n",
+    "Run the following cell to set up code-checking, which will verify your work as you go."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Set up code checking\n",
+    "from learntools.core import binder\n",
+    "binder.bind(globals())\n",
+    "from learntools.machine_learning.ex2 import *\n",
+    "print(\"Setup Complete\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 1: Loading Data\n",
+    "Read the file into a Pandas DataFrame called `home_data`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "# Path of the file to read\n",
+    "iowa_file_path = '../input/house-prices-advanced-regression-techniques/train.csv'\n",
+    "\n",
+    "# Fill in the line below to read the file into a variable home_data\n",
+    "#home_data = _\n",
+    "\n",
+    "# Call line below with no argument to check that you've loaded the data correctly\n",
+    "step_1.check()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Lines below will give you a hint or solution code\n",
+    "# step_1.hint()\n",
+    "# step_1.solution()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 2: Review The Data\n",
+    "Use the command you learned to view summary statistics of the data. Then fill in variables to answer the following questions"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "# Print summary statistics in next line\n",
+    "_"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# What is the average lot size (rounded to nearest integer)?\n",
+    "ave_lot_size = _\n",
```

Comment:
"ave"

Reply:
I'm not sure what this comment is saying.

Reply:
Sorry just pointing out what I think is a typo (should be `avg_lot_size`, right?)

Reply:
(Replying to your response here, since it looks like Github put it as a top-level comment that I can't reply to)

Wow, that's really surprising to me. I didn't even consider the possibility that it wasn't a typo. I don't think I've ever seen "ave" as an abbreviation for "average". Maybe it's another econometrics thing :)